src/index.html

```diff
@@ -10,8 +10,198 @@
+            id="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <br>
+        <label>
+          Full date of birth:
+          <input 
+            type="date" 
+            name="date"
+          >
+        </label>
+        <br>
+        <label>
+          I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            name="acception"
+          >
+        </label>
+      </fieldset>
+    </form>
+
+    <form>
+      <fieldset>
+        <legend>
+          Registration
+        </legend>
+
+        <label>
+          E-mail:
+          <input 
+            type="email" 
+            name="email"
+            placeholder="email@example.com"
+            multiple
+            autocomplete="off"
+            required
+          >
+        </label>
+        <br>
+        <label>
+          Password:
+          <input 
+            type="password" 
+            name="password"
+            minlength="4"
+            maxlength="12"
+            autocomplete="off"
+            required
+          >
+        </label>
+      </fieldset>
+    </form>
+
+    <form>
+      <fieldset>
+        <legend>
+          An interersting fact about you?
+        </legend>
+        <label>
+          Do you love cats?
+          <input
+            type="radio"
+            id="catsChoiseYes"
+            name="love"
+            value="yes"
+          >
+          <label 
+            for="catsChoiseYes"
+          >
+            Yes
+          </label>
+          <input
+            type="radio"
+            id="catsChoiseNo"
+            name="love"
+            value="no"
+          >
+          <label 
+            for="catsChoiseNo"
+          >
+            No
+          </label>
+        </label>
+        <br>
+        <label>
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+          >
+        </label>
+        <br>
+        <label>
+          What time do you go to bed?
+          <input
+            type="time"
+            step="2"
+            name="sleepTime"
+          >
+        </label>
+        <br>
+        <label>
+          What are your favorite brand of cars?
+          <select 
+            name="cars"
+            size="4"
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+            <!-- <option value="none"></option> -->
+          </select>
+        </label>
+        <br>
+        <label>
+          How do you rate our work?
+          <input 
+            type="range"
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>
+          Additional info
+        </legend>
+        <label>
+          Comments:
+          <textarea>
+            name="comments"
+            autocomplete="off"
+          </textarea>
```

Comment:
fix it ![image](https://user-images.githubusercontent.com/91411692/230862449-203838bd-1778-4703-a332-f0c0dbb3c9e5.png)
![image](https://user-images.githubusercontent.com/91411692/230862795-fd166a03-9bfe-4985-ad70-1b2ce65187bb.png)

Reply:
I wrote it in my question in slack, if I write 
<textarea
></textarea>
platform stopped me, because closing tag should be on string with opening tag